rewards/src/lib.rs

```diff
@@ -172,12 +172,14 @@ impl<T: Trait> Module<T> {
 					return;
 				}
 
+				// pay reward to `who`
```

Comment:
can also make `Pools::<T>::mutate(pool, |pool_info| {` to be `try_mutate` and just return the payout result